packages/dara-core/dara/core/internal/encoder_registry.py

```diff
@@ -71,6 +71,15 @@ def _get_pandas_array_encoder(array_type: Type[Any], dtype: Any, raise_: bool =
     )
 
 
+def _tuple_key_serialize(d):
```

Comment:
Can we add a docstring explaining what and why we're doing here for future reference